python/lsst/ts/eas/eas_csc.py

```diff
@@ -278,23 +314,26 @@ async def run_control(self) -> None:
         if self.simulation_mode != 0:
             return
 
-        try:
-            mixing = await self.m1m3ts.tel_mixingValve.next(
-                flush=True, timeout=SAL_TIMEOUT
-            )
-        except Exception:
-            await self.fault(
-                code=THERMAL_LOOP_ERROR,
-                report="Failed to get mixing valve telemetry from M1M3TS.",
-                traceback=traceback.format_exc(),
-            )
+        async with self.control_loop_lock:
```

Comment:
you are just getting startup telemetry here, do you really need to acquire the lock?